app/src/main/java/com/twilio/conversations/app/common/extensions/TwilioExtensions.kt

```diff
@@ -18,57 +17,6 @@ class ConversationsException(val error: ConversationsError, val errorInfo: Error
     constructor(errorInfo: ErrorInfo) : this(ConversationsError.fromErrorInfo(errorInfo), errorInfo)
```

Comment:
Lets rename this like
`ConversationsException` -> `AppException`
`ConversationsError` -> `AppError`

Otherwise it duplicate names with what we have in SDK...
And even after that we have a problem that new SDK extensions now throws `ConversationsException` but in demoapp code we catch `AppException` everywhere... So it basically will not work. This should be fixed.

Reply:
Renamed `ConversationsException` -> `AppException`.

I don't see `ConversationsError` in SDK, see just `enum class Error` there.
Do you still want to rename `ConversationsError` -> `AppError`?